doc/user/content/sql/create-materialized-view.md

```diff
@@ -1,78 +1,92 @@
+description: "`CREATE MATERIALIZED VIEW` defines a view that is persisted in storage and incrementally updated as new data arrives."
 menu:
   main:
     parent: 'commands'
 ---
 
-`CREATE MATERIALIZED VIEW` creates a materialized view, which lets you retrieve
-incrementally updated results of a `SELECT` query very quickly. Despite the
-simplicity of creating a materialized view, it's Materialize's most powerful
-feature.
+`CREATE MATERIALIZED VIEW` defines a view that is persisted in storage and
+incrementally updated as new data arrives.
 
-## Conceptual framework
-
-`CREATE MATERIALIZED VIEW` computes and maintains the results of a `SELECT`
-query in memory. For more information, see [Key Concepts: Materialized views](/overview/key-concepts/#materialized-views).
+A materialized view specifies a [cluster](/overview/key-concepts/#clusters) that
+is tasked with maintaining its results up-to-date, but **can be referenced in
+any cluster**. This allows you to effectively decouple the computational
+resources used for view maintenance from the resources used for query serving.
 
 ## Syntax
 
 {{< diagram "create-materialized-view.svg" >}}
 
 Field | Use
 ------|-----
-**OR REPLACE** | If a view exists with the same name, replace it with the view defined in this statement. You cannot replace views that other views depend on, nor can you replace a non-view object with a view.
-**IF NOT EXISTS** | If specified, _do not_ generate an error if a view of the same name already exists. <br/><br/>If _not_ specified, throw an error if a view of the same name already exists. _(Default)_
-_view&lowbar;name_ | A name for the view.
+**OR REPLACE** | If a materialized view exists with the same name, replace it with the view defined in this statement. You cannot replace views that other views depend on, nor can you replace a non-view object with a view.
+**IF NOT EXISTS** | If specified, _do not_ generate an error if a materialized view of the same name already exists. <br/><br/>If _not_ specified, throw an error if a view of the same name already exists. _(Default)_
+_view&lowbar;name_ | A name for the materialized view.
 **(** _col_ident_... **)** | Rename the `SELECT` statement's columns to the list of identifiers, both of which must be the same length. Note that this is required for statements that return multiple columns with the same identifier.
-_cluster&lowbar;name_ | The cluster to maintain this materialized view. If not provided, uses the session’s cluster variable.
-_select&lowbar;stmt_ | The [`SELECT` statement](../select) whose output you want to materialize and maintain.
+_cluster&lowbar;name_ | The cluster to maintain this materialized view. If not specified, defaults to the active cluster.
+_select&lowbar;stmt_ | The [`SELECT` statement](../select) whose results you want to maintain incrementally updated.
 
 ## Details
 
-### Memory
+### Usage patterns
```

Comment:
For core concepts, I think it's important to sneak in a little more meat than just providing a syntax reference. This is similar to what we're doing with the `CREATE SOURCE` reference pages.

This should be supported by more detailed operational guides that we can point to from these barer explanations.